Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Resources_Leathers.xml

```diff
@@ -2,313 +2,274 @@
-				</value>
-			</li>
-
-			<li Class="PatchOperationRemove">
-				<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases/SharpDamageMultiplier</xpath>
-			</li>
-
-			<li Class="PatchOperationAdd">
-				<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/stuffProps/categories</xpath>
-				<value>
-					<li>Metallic_Weapon</li>
-					<li>Steeled</li>
-				</value>
-			</li>
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases/StuffPower_Armor_Sharp</xpath>
-				<value>
-					<StuffPower_Armor_Sharp>1.2</StuffPower_Armor_Sharp>
-				</value>
-			</li>
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases/StuffPower_Armor_Blunt</xpath>
-				<value>
-					<StuffPower_Armor_Blunt>1.6</StuffPower_Armor_Blunt>
-				</value>
-			</li>
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases/StuffPower_Armor_Heat</xpath>
-				<value>
-					<StuffPower_Armor_Heat>-0.05</StuffPower_Armor_Heat>
-				</value>
-			</li>
-			
-			<!--aerofleet stuff and plant leather -->
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_Leather_Aerofleet" or defName="Leather_Cactus"]/statBases/StuffPower_Armor_Sharp</xpath>
-				<value>
-					<StuffPower_Armor_Sharp>0.03</StuffPower_Armor_Sharp>
-				</value>
-			</li>
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_Leather_Aerofleet"]/statBases/StuffPower_Armor_Blunt</xpath>
-				<value>
-					<StuffPower_Armor_Blunt>0.032</StuffPower_Armor_Blunt>
-				</value>
-			</li>
-
-			
-			<li Class="PatchOperationAdd">
-				<xpath>/Defs/ThingDef[defName="Leather_Cactus"]/statBases</xpath>
-				<value>
-					<StuffPower_Armor_Blunt>0.032</StuffPower_Armor_Blunt>
-				</value>
-			</li>
-			
-			<!-- Lion Leather -->
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_ArcticLion_Leather" or defName="AA_SandLion_Leather"]/statBases/StuffPower_Armor_Sharp</xpath>
-				<value>
-					<StuffPower_Armor_Sharp>0.08</StuffPower_Armor_Sharp>
-				</value>
-			</li>	
-
-			<li Class="PatchOperationAdd">
-				<xpath>/Defs/ThingDef[defName="AA_ArcticLion_Leather" or defName="AA_SandLion_Leather"]/statBases</xpath>
-				<value>
-					<StuffPower_Armor_Blunt>0.06</StuffPower_Armor_Blunt>
-				</value>
-			</li>
-			
-			<!-- Night Leather. It was made through Conditionals to allow Forsaken's compatibility -->
-			
-			<li Class="PatchOperationConditional">
-			<xpath>/Defs/ThingDef[defName="Leather_Night"]/statBases/StuffPower_Armor_Sharp</xpath>
-				<nomatch Class="PatchOperationAdd">
-				<xpath>/Defs/ThingDef[defName="Leather_Night"]/statBases</xpath>
+					</value>
+				</li>
+				<!-- SkySteel -->
+				<li Class="PatchOperationAdd">
+					<xpath>/Defs/ThingDef[defName="AA_SkySteel"]/statBases</xpath>
 					<value>
-						<StuffPower_Armor_Sharp>0.08</StuffPower_Armor_Sharp>
+						<Bulk>0.025</Bulk>

```

Comment:
Dude, can you edit files cleaner? You managed to change 539 lines and possibly f*cked up a few values just to add some stats to wool.

Reply:
I Refactored the xml lines while adding the mouflon wool at the end, nothing was fucked up doing so.

Reply:
Then sorry. But I'd advice you to refactor files before the changes as a separate commit. Keeps things cleaner.

Reply:
Yeah I should have done that...